kotlinx-knit-test/build.gradle.kts

```diff
@@ -10,4 +12,13 @@ val dokka by tasks.getting(DokkaTask::class) {
 val dokkaHtml by tasks.creating(DokkaTask::class) {
     outputFormat = "html"
     outputDirectory = "$buildDir/dokkaHtml"
-}
\ No newline at end of file
+}
+
+publishing {
+    publications {
+        create<MavenPublication>("kotlinxKnitTest") {
```

Comment:
Could you please elaborate on what's the point ofpublishing these?

Reply:
It is needed for knit to run (see the dependency to `implementation(project(":kotlinx-knit-test"))` in the root `build.gradle` file), so I actually don't know why it's not published. This is just a basic publication that I needed for local testing, probably should be rewritten, but I don't know how do you plan on publishing it, that's why I left it this way